app/views/landing-pages/parents/PageParents.vue

```diff
@@ -26,851 +27,853 @@
+            <div class="col-lg-12 trust-logos">
+              <div class="flex-spacer">
                 <img
-                  src="/images/pages/parents/cs_for_all_member.png"
+                  src="/images/pages/parents/cse_top_pick.png"
+                  class="cse-top-pick"
                 >
               </div>
-            </div>
 
-            <div class="flex-spacer">
-              <div class="codie-logo-container">
-                <img
-                  src="/images/pages/parents/2017_codie_award.png"
-                >
+              <div class="flex-spacer">
+                <div class="cs-for-all-container">
+                  <img
+                    src="/images/pages/parents/cs_for_all_member.png"
+                  >
+                </div>
+              </div>
+
+              <div class="flex-spacer">
+                <div class="codie-logo-container">
+                  <img
+                    src="/images/pages/parents/2017_codie_award.png"
+                  >
+                </div>
               </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
 
-    <div class="container-graphic-spacer sm-min-height-auto blue-fox-spacer" />
+      <div class="container-graphic-spacer sm-min-height-auto blue-fox-spacer" />
 
-    <div class="container">
-      <div class="row">
-        <h1
-          class="text-center pixelated"
-          style="padding: 0 5px;"
-        >
-          {{ $t('parents_landing_1.remote_learning_works') }}
-        </h1>
-        <div class="col-xs-12 video-container">
-          <div style="position: relative; padding-top: 56.25%;">
-            <iframe
-              :src="'https://iframe.videodelivery.net/' + videoId + '?preload=true&poster=https://videodelivery.net/' + videoId + '/thumbnails/thumbnail.jpg%3Ftime%3D2s&defaultTextTrack=en'"
-              style="border: none; position: absolute; top: 0; height: 100%; width: 100%;"
-              allow="accelerometer; gyroscope; autoplay; encrypted-media; picture-in-picture;"
-              allowfullscreen="true"
-              title="CodeCombat online classes video"
-            />
-          </div>
-        </div>
-      </div>
-    </div>
-
-    <div class="container-graphic-spacer">
       <div class="container">
         <div class="row">
-          <div
-            class="col-xs-12"
-            style="margin: 30px 0 20px;"
+          <h1
+            class="text-center pixelated"
+            style="padding: 0 5px;"
           >
-            <img
-              class="img-responsive"
-              src="/images/pages/parents/graphic_09.png"
-              alt="hero moving along a path"
-              loading="lazy"
-              style="max-width: 290px;"
-            >
+            {{ $t('parents_landing_1.remote_learning_works') }}
+          </h1>
+          <div class="col-xs-12 video-container">
+            <div style="position: relative; padding-top: 56.25%;">
+              <iframe
```

Comment:
why are there changes on existing page? Most look like eslint or something

Reply:
In order to make it possible to inherit the modals (for CTA buttons), I had to add a `<slot`, so everything indented.